core/src/main/java/org/apache/accumulo/core/client/impl/Namespace.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.core.client.impl;
+
+import org.apache.accumulo.core.client.Instance;
+
+public class Namespace {
```

Comment:
> Is there some kind of longer-term goal with what the Namespace and Table classes will become?

Yes this name came from ACCUMULO-3238 and the (unfinished) work @ctubbsii did for ACCUMULO-2589.  Basically, creating the opportunity for more useful Table and Namespace objects, while improving type safety in the meantime.  Plus there is already org.apache.accumulo.core.data.TabletId